### PR TITLE
Fut 44 add button save

### PR DESCRIPTION
**Pull request by karollayna**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -20,7 +20,9 @@ Imports:
     magrittr,
     utils,
     ggplot2,
-    plotly
+    plotly,
```
Comment (darsoo) [inline DESCRIPTION:23]: sort please

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+0.0.18
+
+- added button: save
+- added loading and saving tables to files
```
Comment (darsoo) [inline NEWS.md:4]: info about change layout?

File: tests/testthat/test_runFutureApp.R [hunk 3]
```diff
@@ -6,6 +6,7 @@ app$waitForShiny()
 
 app$click("main_module-add_one")
 app$waitForShiny()
+Sys.sleep(1)
```
Comment (darsoo) [inline tests/testthat/test_runFutureApp.R:9]: please add comment why

File: inst/shiny/tests/shinytest/test_runFutureApp-current/002.json [hunk 4]
```diff
@@ -103,9 +98,9 @@
               "text"
             ],
             "text": [
-              "8.2  gram",
```
Comment (darsoo) [inline inst/shiny/tests/shinytest/test_runFutureApp-current/002.json:106]: should this change?

File: R/modules.R [hunk 5]
```diff
@@ -138,27 +187,77 @@ mainModule <- function(input, output, session){
...
+
+               colnames(info_about_prepared_meal)[[1]] <- "meal_name"
+
+               rv$ingreadients_of_meal <-
+                  rbind(rv$ingreadients_of_meal,
+                        info_about_prepared_meal)
+
+               utils::write.table(
+                  rv$meals_table,
+                  file = file.path(system.file(package = "Future"), "meals.tsv"),
```
Comment (darsoo) [inline R/modules.R:231]: add TODO with a possibility to change it

**Review by darsoo — APPROVED** [review]

File: inst/shiny/tests/shinytest/test_runFutureApp-current/002.json (hunk 4 above)
Comment (karollayna) [reply inst/shiny/tests/shinytest/test_runFutureApp-current/002.json:106]: yes, previously was incorrect

**Pull request merged**